pubspec.yaml

```diff
@@ -22,22 +22,29 @@ environment:
 
 dependencies:
   cupertino_icons: ^1.0.2
+  dio: ^4.0.0
   flutter:
     sdk: flutter
   flutter_localizations:
     sdk: flutter
   flutter_config: ^2.0.0
+  freezed: ^0.14.1+3
+  freezed_annotation: ^0.14.1
+  injectable: ^1.3.0
```

Comment:
it seems we are missing `get_it` and `injectable_generator` packages. [Reference](https://pub.dev/packages/injectable#installation) 😅

Reply:
@manh-t we don't implement the DI part in this task so adding get_it and injectable_generator are unnecessary

Reply:
@luongvo shall we remove `network_module`? It's getting error there 💅

Reply:
@manh-t agree to remove